src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -507,6 +514,9 @@ public enum ArgumentId
 
             // Request a server shutdown from the client
             Shutdown,
+
+            // The directory to use for temporary operations.
+            TempDirectory,
```

Comment:
I think it should probably be deleted.  It's not even relevant with the way we changed the pipe names.  Let me think about it for a sec.